spec/datetimeformat.html

```diff
@@ -364,14 +364,25 @@ <h1>FormatDateTimeToParts ( _dateTimeFormat_, _x_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-tolocaltime" aoid="ToLocalTime">
-      <h1>ToLocalTime ( _date_, _calendar_, _timeZone_ )</h1>
+      <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
 
       <p>
-        When the ToLocalTime abstract operation is called with arguments _date_, _calendar_, and _timeZone_, the following steps are taken:
+        When the ToLocalTime abstract operation is called with arguments _t_, _calendar_, and _timeZone_, the following steps are taken:
       </p>
 
       <emu-alg>
-        1. Apply calendrical calculations on _date_ for the given _calendar_ and _timeZone_ to produce weekday, era, year, month, day, hour, minute, second, and inDST values. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules. If the _calendar_ is `"gregory"`, then the calculations must match the algorithms specified in ES2020, <emu-xref href="#sec-overview-of-date-objects-and-definitions-of-abstract-operations"></emu-xref>.
+        1. Assert: Type(_t_) is Number.
+        1. Apply calendrical calculations on _t_ for the given _calendar_ and _timeZone_ to produce weekday, era, year, month, day, hour, minute, second, and inDST values. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules.
```

Comment:
How about something more like,

```
1. Assert: Type(_t_) is Number.
1. If _calendar_ is `"gregory"`,
  1. Return a Record with fields calculated according to <reference to table>
1. Else,
  1. Return a Record with the fields in Column 1 of <reference to table>, calculated by (insert hand-wavy prose here).
```

Reply:
LGTM, @spectranaut ?

Reply:
Looks good to me -- I'll use this structure, but I'm confused what table you mean to be referencing, @sffc ? There is no table with just the entries of this record we are constructing that I can find..

Reply:
I'm suggesting that you make a table similar to https://github.com/tc39/proposal-intl-displaynames/pull/44/files and https://github.com/tc39/proposal-unified-intl-numberformat/pull/58.  Something like,

| Field | Algorithm for calendar == `"gregory"` |
|---|---|
| `[[weekday]]` | The result of `WeekDay(tz)` specified in ES2020's <emu-xref href="#sec-week-day">Week Day</emu-xref>. |

and so forth.

Reply:
nice. will do!